src/controller/python/matter/commissioning/pase.py

```diff
@@ -50,10 +50,14 @@ async def establish_session(devCtrl: ChipDeviceCtrl.ChipDeviceControllerBase, pa
     elif isinstance(parameter, commissioning.PaseOverIPParameters):
         device = await devCtrl.DiscoverCommissionableNodes(filterType=discovery.FilterType.LONG_DISCRIMINATOR,
                                                            filter=parameter.long_discriminator, stopOnFirst=True)
+        selected_address = None
         if not device:
             raise ValueError("No commissionable device found")
-        selected_address = None
-        for ip in device[0].addresses:
+        if isinstance(device, list):
```

Comment:
DiscoverCommissionableNodes always returns a list, right? So this should not be needed. Is the type hint for this incorrect or something?

Reply:
Is the type hint incorrect? I don't see where it returns none, it returns an empty list in the case of no discovered devices.